### DISCUSSION TITLE
Does Mission Scrub recommendation have any effect?

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#589] [Q&A] [answered]
I was wondering today about when you have a non-catastrophic failure that leads Mission Control to recommend a scrub.  Is that just an attempt to spook you, or does continuing against a recommendation give some kind of penalty for later steps of the mission?

**Comment by hweimer** [comment, marked as answer]
Not really a penalty for continuing, but the failure temporarily damages the hardware  by 1-20% (independent of whether you continue or not), so you might want to play it safe nevertheless.

**Reply by peyre** [reply]
Ah!  So there is some game-mechanic difference.  It isn't just a situation where Mission Control is skittish but you face the same percentages - there is some safety penalty to going against MC's recommendations.

If so, should we mention it in the mission fail text, or should it be mentioned in the manual?  The player _should_ be aware that there is some penalty, in game terms, to proceeding against recommendations.

**Reply by peyre** [reply]
Hm, on a closer look this seems a little tricky.  I don't see a convenient place to mention this in the manual.  It could go comfortably in my RIS guide, but that doesn't make it obvious to the player that there's an in-game difference when a scrub is recommended.  And sticking it in fails.cdr isn't so simple either: it would need to be done in multiple places, and it might be hard to mention without breaking immersion.

I wonder if I could insert it into the Step Failure screen...

**Reply by peyre** [reply]
Ok, got it!  The Step Failure screen now warns the player there will be a temporary penalty on the hardware.
